e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -203,6 +203,41 @@ describe('Interactive Menu', () => {
         });
     });
 
+    it('IM21043 - Using up/down arrow keys to make selection', () => {
+        const basicOptions = getMessageMenusPayload({options});
+
+        // # Post an incoming webhook for interactive menu with basic options
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: basicOptions});
+
+        // # Get message attachment from the last post
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#messageAttachmentList_${postId}`).as('messageAttachmentList');
+        });
+
+        cy.get('@messageAttachmentList').within(() => {
+            cy.get('.select-suggestion-container > input').click();
```

Comment:
We're slowly adopting getting elements based on how the "real" user sees it.  So, instead of using above selector, let's get it by its placeholder.  Also, since it's being used multiple times, you may use alias.
```suggestion
            cy.findByPlaceholderText('Select an option...').as('optionInputField');
            cy.get('@optionInputField').click();
```

Reply:
Thank you for this feedback. I'll add the changes.